ts/smelter-node/src/manager/locallySpawnedInstance.ts

```diff
@@ -82,10 +91,39 @@ class LocallySpawnedInstanceManager implements SmelterManager {
 
     await retry(async () => {
       await sleep(500);
-      return await this.sendRequest({
+
+      let status = (await this.sendRequest({
```

Comment:
wrap request in some function, convert imiedietlly to cammel case types and write all logic while operating on those values